docs/src/how-to/administrate/users.rst

```diff
@@ -607,3 +607,40 @@ Where:
 * `$EMAIL_CODE` is the validation code received by email after running the previous script/command
 * `$TEAM_CURRENCY` is the currency of the team
 * `$TEAM_NAME` is the name of the team
+
+Enabling debugging for backend services
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, backend services (pods) might not output as much information as needed to resolve specific issues.
+
+You can change this for any of the services by editing its configuration.
+
+For example, to enable debugging for the `cargohold` service, you can edit the `cargohold` configuration in your ``values.yaml``:
+
+.. code:: yaml
+
+   cargohold:
+      replicaCount: 3
+      config:
+         logLevel: Debug
+         aws:
+            s3Bucket: prod-assets-test
+            s3Endpoint: https://s3.us-east-1.amazonaws.com
+            s3DownloadEndpoint: https://s3.us-east-1.amazonaws.com
+
+The possible values for ``logLevel`` are:
+
+* ``Debug``
+* ``Info``
+* ``Warn``
+* ``Error``
+* ``Fatal``
+* ``Trace``
```

Comment:
Would it be worth exposing here what each of these levels mean?

Reply:
At least, I would order them by severeness and explain that some of them may leak secret information (`DEBUG` and `TRACE`).

Reply:
what is the order?

Reply:
```
/nix/store/vdqarafm2zrvw7892cvi4gq540b8d07q-tinylog-0.15.0-doc/share/doc/tinylog-0.15.0/html/System-Logger.html: openFile: does not exist (No such file or directory)
```